CONTRIBUTING.md

```diff
@@ -83,17 +114,170 @@ In general, pull requests use `develop` as the base branch.
 Changes to pull requests must be added as new commits.
 Once code reviewers have started looking at your code, please avoid
 force-pushing a branch in a pull request.
-This preserves the ability for reviewers to filter changes since their last
-review.
+This preserves the ability for reviewers to filter changes since their
+last review.
 
-A pull request must obtain **approvals from at least two reviewers** before it
-can be considered for merge by a Maintainer.
+A pull request must obtain **approvals from at least two reviewers**
+before it can be considered for merge by a Maintainer.
 Maintainers retain discretion to require more approvals if they feel the
 credibility of the existing approvals is insufficient.
 
```

Comment:
Once a pull request has been marked as "Ready" (i.e. no longer a draft), please do not rebase it on other branches, and use a merge operation instead.